### PR TITLE
[test](nereids) Add some test for single table materialized view rewrite without aggregate

### PR DESCRIPTION
**Pull request by seawinde**
## Proposed changes

Add some test for single table materialized view rewrite without aggregate
Support sql pattern as following

classify each type test with the number prefix in code
1-
select + from
2-
select + from + order by + limit
3-
select + from + filter
4-
select + from + filter + order by + limit
5-
select + from + sub query(select + from)
select + from + sub query(select + from + order by + limit)
select + from + sub query(select + from + filter)
select + from + sub query(select + from + filter + order by + limit)
6-
select + from + view(select + from)
select + from + view(select + from + order by + limit)
select + from + view(select + from + filter)
select + from + viewselect + from + filter + order by + limit)
7-
cte(select + from) + select + from 
cte(select + from + order by + limit) + select + from 
cte(select + from + filter) + select + from
cte(select + from + filter + order by + limit) + select + from

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by seawinde** [comment]
run buildall

**Comment by seawinde** [comment]
run buildall

**Comment by seawinde** [comment]
run buildall

File: regression-test/suites/nereids_rules_p0/mv/single_table_without_aggregate.groovy [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    sql """drop view if exists ${view6_3_name}"""
+    sql """
+         create view ${view6_3_name} as ${query4_1}
+    """
+    def query6_3 = "select O_ORDERKEY, count(*) from " +
+            "${view6_3_name} " +
+            "group by O_ORDERKEY"
+    check_rewrite(mv4_1, query6_3, "mv6_3")
+    order_qt_query6_3 "${query6_3}"
+    sql """DROP MATERIALIZED VIEW IF EXISTS mv6_3 ON orders"""
```
Comment (morrySnow) [inline regression-test/suites/nereids_rules_p0/mv/single_table_without_aggregate.groovy:271]: cte

**Comment by seawinde** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Comment by seawinde** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Pull request closed without merging**